OptionalCommand/Jackalope/InitDoctrineDbalCommand.php

```diff
@@ -47,7 +48,17 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        DoctrineCommandHelper::setApplicationConnection($this->getApplication(), $input->getOption('session'));
+        $application = $this->getApplication();
+        $sessionName = $input->getOption('session');
+        if (empty($sessionName)) {
+            $sessionName = DoctrineCommandHelper::getAdminSession($application) ?: $application->getKernel()->getContainer()->hasParameter('doctrine_phpcr.default_session');
+
+            if (empty($sessionName)) {
+//                throw new RuntimeException("No session name could be determined. This command requires either that the 'session' option is passed to the command or an 'admin_session' is configured in the DoctrinePHPCRBundle configuration, with a workspace name that must exist ('default' is recommended)");
```

Comment:
if we wouldn't use ``default_session`` as a fallback we could enable this .. but it would wreck havoc on our test setup again and it might also confuse users needlessly .. so I guess we can remove this .. wdyt?